src/i_sound.c

```diff
@@ -91,16 +91,11 @@ static void StopChannel(int channel)
     return;
 #endif
 
-  if (channelinfo[channel].data)
+  if (channelinfo[channel].enabled)
   {
     alSourceStop(openal_sources[channel]);
 
-    // [FG] immediately free samples not connected to a sound SFX
-    if (channelinfo[channel].sfx == NULL)
-    {
-      free(channelinfo[channel].data);
```

Comment:
Do we ever release sound on the old system? It seems to be always cached.

Reply:
Yes, we immediately release buffers which contain pitch-shifted sounds. But this concept doesn't hold anymore. 😉

Reply:
So, should we? Doom sfx takes very little memory by today's standards.

Reply:
I'd like to keep a cached copy of each sound, just as we are doing now. I am not concerned about their size and and benefit of not having to reload them from the WAD and convert them target format clearly outweights the few KB of RAM it costs. Also, we don't have pitch-shifted sampels in RAM anymore, because we just tell OpenAL to play a sound pitch-shifted on a per-channel basis. So, to me it's good as it is now.

Or did you actually mean anything different?